mubench.cli/README.md

```diff
@@ -41,3 +42,26 @@ It supports two run modes:
 In either mode, the `DetectorArgs` provide all input as both the Java source code and the corresponding Bytecode. Additionally, it provides the classpath of all the code's dependencies.
 
 The `DetectorOutput` is essentially a collection where you add your detector's findings, specifying their file and method location and any other property that may assist manuel reviews of the findings. MUBench expects you to add the findings ordered by the detector's confidence, descending. You may also output general statistics about the detector run.
+
+## List of Detector Releases
+
+This file must be at `detectors/your-detector/releases.yml` and contains a list of releases of your detector.
+
+Entries look like this:
+
+    - cli-version: 0.0.10
+      md5: 2470067fac02ee118b7d49287246e20a
+    - tag: icse17
+      cli-version: 0.0.8
+      md5: 9e5252816faecf552464f0a6abde714f
+
+The list should be sorted by newest at the top, oldest at the bottom. By default, MUBench will use the first version listed.
+`tag` is optional and can be used to reference specific detector releases. `cli-version` is the [MUBench Runner](#runner) version implemented by the respective detector release. `md5` must match the md5 of the `detector/my-detector/my-detector.jar`.
+
+## Detector.py
+
+To configure your detector on the python side, you need to create `detectors/your-detector/your-detector.py`.
+
+This script should implement a subclass of `data.detector.Detector`, which must define `_specialize_finding(self, findings_path: str, finding: Finding) -> SpecializedFinding`. A specialized finding is supposed to convert the output of the detector to human-readable output, which will be displayed on the review page. The `data.detector_specialising.specialising_util` module contains utilities for this purpose.
```

Comment:
Please link `data.detector.Detector` to the respective .py file implementing it. Same for the specialising_util.